### PR TITLE
Feature/article page

### PR DESCRIPTION
**Pull request by Majid-pkz**

### CODE REVIEW

**Comment by Majid-pkz** [comment]
basic style added. will work on it tomorrow

File: src/app/components/pages/articleData.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "articles": [
+      {
+        "id": 1,
+        "header": "Assistive Technology",
+        "imageSrc": "https://picsum.photos/200/300",
```
Comment (bhumkarpratik) [inline src/app/components/pages/articleData.json:6]: You could set the actual size here.. @Majid-pkz

File: src/app/components/pages/TextHeavyArticle.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}: TextHeavyInterface): JSX.Element {
+  return (
+    <div>
+      <Typography
+        variant={TypographyVariant.Body2}
+      >{`Home > Articles > ${header}`}</Typography>
+      <Typography variant={TypographyVariant.H2}>{header}</Typography>
+      <Image width={1150} height={612} src={imageSrc} alt={header} />
+      <div>
+        <Typography variant={TypographyVariant.Body2}>{bodyText}</Typography>
```
[the remark this answers is not in the document]
Comment (bhumkarpratik) [reply src/app/components/pages/TextHeavyArticle.tsx:20]: Hey I have figured out why it comes up as bold.
Please delete lines 12 to 15 from global.css

**Review by bhumkarpratik — COMMENTED** [review]
There must be more padding top and bottom of the text and it should be better.
The breadcrumbs should redirect the user to the homepage to read more articles.

**Comment by Majid-pkz** [comment]
my observation :
1. Text is not justified in all views
2.  gap between the text and photo could be more in desktop view
1. breadcrumb font size for mobile view is too large (only for mobile type- screen smaller than 360)

**Pull request merged**